Mathlib/Analysis/InnerProductSpace/Symmetric.lean

```diff
@@ -130,10 +130,12 @@ theorem IsSymmetric.restrict_invariant {T : E →ₗ[𝕜] E} (hT : IsSymmetric
     (hV : ∀ v ∈ V, T v ∈ V) : IsSymmetric (T.restrict hV) := fun v w => hT v w
 #align linear_map.is_symmetric.restrict_invariant LinearMap.IsSymmetric.restrict_invariant
 
+@[nolint docBlame defLemma] -- linter bug?
+local instance : ContinuousSMul ℝ 𝕜 := inferInstance in
+@[nolint docBlame] -- linter bug?
+noncomputable local instance : InnerProductSpace ℝ E := InnerProductSpace.isROrCToReal 𝕜 E in
 theorem IsSymmetric.restrictScalars {T : E →ₗ[𝕜] E} (hT : T.IsSymmetric) :
```

Comment:
Investigate

Reply:
[This](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/defLemma.20and.20docBlame.20linters.20on.20local.20instances.20in.20sections/near/433824150) may be related.